packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -61,6 +61,7 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
   RemoteSecondary? _remoteSecondary;
   AtClientCommitLogCompaction? _atClientCommitLogCompaction;
   AtClientConfig? _atClientConfig;
+  static var upperCaseRegex = RegExp(r'[A-Z]');
```

Comment:
static const please

Reply:
dart's not allowing me to make it a static const, it needs the RegExp constructor to be a const too(which it is not). Does making in static final work instead of static const ?

ps: the actual error being displayed by dart: 'The constructor being called isn't a const constructor.'

Reply:
oh yeah I forgot that RegExp doesn't have a const constructor, because it can throw an exception

so just just static final